### PR TITLE
Slice Machine part 3

### PR DESCRIPTION
**Pull request by davidpmccormick**
These were generated from our existing types using `@slicemachine/init`

### CODE REVIEW

File: common/customtypes/visual-stories/index.json [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+            },
+            "textAndImage": {
+              "type": "Slice",
+              "fieldset": "Text and image",
+              "description": "Side-by-side",
+              "non-repeat": {
+                "text": {
+                  "type": "StructuredText",
+                  "config": {
+                    "multi": "paragraph,hyperlink,strong,em,heading3",
```
Comment (rcantin-w) [inline common/customtypes/visual-stories/index.json:158]: add list-item

Comment (rcantin-w) [reply common/customtypes/visual-stories/index.json:158]: add it to text and icons as well

**Review by rcantin-w — COMMENTED** [review]
Can you remind me how these are generated? Are they what we'd use to replicate our slices to a new project?

**Comment by davidpmccormick** [comment]
These were generated using `@slicemachine/init` which pulled down what we had and generated the appropriate shape JSON for a Slice Machine project. They are what we'd push to the Prismic staging environment in order to test our Slice Machine work with data that is in the same shape as what we'll ultimately be using

**Review by rcantin-w — APPROVED** [review]

**Pull request merged**